ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -43,6 +43,19 @@ def AddV2(*, x, y, name="AddV2"):
     return ivy.addv2(tf_frontend.to_tensor(x), tf_frontend.to_tensor(y))
 
 
+@to_ivy_arrays_and_back
+def ApproximateEqual(
```

Comment:
there is a list of limited accepted `dtypes` for this function, please check the docs for more info: https://www.tensorflow.org/api_docs/python/tf/raw_ops/ApproximateEqual

then add the `@with_unsupported_dtypes` accordingly